src/test/javascript/integration/historical_earliest_item_in_channel_spec.js

```diff
@@ -32,11 +31,13 @@ describe(testName, function () {
         var historicalItem2 = channelResource + '/' + '2013/11/20/12/01/00/000';
         utils.postItemQ(historicalItem1)
             .then(function (value) {
-                items.push(value.response.headers.location);
+                const location = fromObjectPath(['response', 'headers', 'location'], value);
+                items.push(location);
                 return utils.postItemQ(historicalItem2);
             })
-            .then(function (value) {
-                items.push(value.response.headers.location);
+            .then(function (nextValue) {
```

Comment:
Is there a reason you're calling this argument `nextValue` instead of just `value`? Same with `nextLocation` vs. `location`?

Again, just curious ;)

Reply:
just a preference, some linters will puke on using the same var name within a larger scope even though they are obviously within their own sub-scope. I just think it reads well to name them differently if they are in the same chain.

Reply:
Ah, gotcha. Works for me. You own this stuff now so whatever stylistically go with whatever you like :)